parcel_test.go

```diff
@@ -31,57 +31,121 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	if err != nil {
+		require.NoError(t, err)
+	}
+	defer db.Close()
+
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
-
+	number, err := store.Add(parcel)
+	if err != nil {
+		require.NoError(t, err)
+	}
+	require.NotEmpty(t, number)
 	// get
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
-
+	storedParcel, err := store.Get(number)
+	if err != nil {
+		require.NoError(t, err)
+	}
+	parcel.Number = number
+	require.Equal(t, storedParcel, parcel)
 	// delete
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что посылку больше нельзя получить из БД
+	err = store.Delete(number)
+	if err != nil {
+		require.NoError(t, err)
+	}
+	_, err = store.Get(number)
+	require.Equal(t, err, sql.ErrNoRows)
```

Comment:
При работе с ошибками лучше использовать специализированные функции
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```